src/fu-engine.c

```diff
@@ -2800,14 +2800,16 @@ fu_engine_get_upgrades (FuEngine *self, const gchar *device_id, GError **error)
 			g_set_error (error,
 				     FWUPD_ERROR,
 				     FWUPD_ERROR_NOTHING_TO_DO,
-				     "No upgrades for device, current is %s: %s",
+				     "No upgrades for %s, current is %s: %s",
```

Comment:
Do we need ``%s'`? Some device names have spaces.

Reply:
I tested with a device titled: `XPS 9370 Thunderbolt controller` and it worked OK.

> "No upgrades for XPS 9370 Thunderbolt Controller, current is 28.0: 28.00=same"

A bigger concern would be if "," was in the name, maybe that would make the string look funny.

Reply:
Fair enough, thanks for the example.